### DISCUSSION TITLE
How to install Debian 12 android box X96 x6

### DISCUSSION THREAD

**Discussion by illlumin** [devmfc/debian-on-amlogic#78] [Q&A]
How to install Debian on a device with the following configuration?

CPU: Rockchip RK3566 Quad Core ARM Cortex A55
GPU: Mali-G52 1-Core-2EE, supports OpenGL ES 1.1/2.0/3.2, OpenCL 2.0, Vulkan 1.1
NPU: Neural network acceleration engine with processing performance up to 0.8 TOPS
RAM: 8GB
Internal Storage: 64GB
Your guidance is greatly appreciated.

**Comment by devmfc** [comment]
I wonder... This repository is called debian-on-amlogic, not debian-on-rockchip, why would you think you would get an answer about a Rockchip box here?
